t/test-data/Models/hr-inclNOxExtendedWithFilters/Storage.nhd

```diff
@@ -106,7 +106,7 @@ gui      = Storage,Hofdüngerlager,Stockage,Storage
   ++description
     Annual NH3 emission from liquid storage of cattle slurry.
   ++formula
-    Sum(nh3_nliquid_cattle, Storage::Slurry);
+    scale( Val(n_into_storage_liquid_cattle_ratio, Livestock), Sum(nh3_nliquid_cattle, Storage::Slurry) );
 
```

Comment:
Hast du getestet, dass das funktioniert? Ich **vermute**, dass das zweite Argument von scale() automatisch in einen Scalar umgewandelt wird (also das Total genommen wird), habe es aber selbst noch nicht ausprobiert.
Ev wäre es zur Klarheit besser, diesen Parameter in ein scalar() zu packen?

Reply:
Ja das habe ich getestet, das zweite Argument wird automatisch in ein Scalar umgewandelt. Gebe dir recht, dass es besser wäre die Funktion scalar zu verwenden. Ich würde dies allerdings gerne später einmal korrigieren, ich muss sowieso nochmals über die nhd Files und die Description etc anpassen & würde dann auch gleich den Code/Syntax wo nötig verbessern...